app.js

```diff
@@ -225,37 +229,32 @@ app.use((req, res) => {
   });
 });
 
-module.exports = {
-  app,
-  version: appVersion
-};
-
 /* istanbul ignore if */
-if (require.main === module) {
-  const {argv} = require('yargs').command(
-      '$0',
-      'Run the mdn-bcd-collector server',
-      (yargs) => {
-        yargs
-            .option('https-cert', {
-              describe: 'HTTPS cert chains in PEM format',
-              type: 'string'
-            })
-            .option('https-key', {
-              describe: 'HTTPS private keys in PEM format',
-              type: 'string'
-            })
-            .option('https-port', {
-              describe: 'HTTPS port (requires cert and key)',
-              type: 'number',
-              default: 8443
-            })
-            .option('port', {
-              describe: 'HTTP port',
-              type: 'number',
-              default: process.env.PORT ? +process.env.PORT : 8080
-            });
-      }
+if (esMain(import.meta)) {
+  const {argv} = yargs(hideBin(process.argv)).command(
```

Comment:
Is this change involving `hideBin` required here, or can it be done separately? Doesn't just `yargs.command(...)` continue to work here?

Reply:
This was taken directly from the Yargs example.  All of the examples, including CommonJS examples, have this new structure so I think there's a good reason for it.

Reply:
Fair enough, I guess it does something useful :)